enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleWidget/MetabaseLinksToggleDescription.tsx

```diff
@@ -0,0 +1,30 @@
+import { t, jt } from "ttag";
+import { Anchor, Popover, Stack, Text } from "metabase/ui";
+
+export function MetabaseLinksToggleDescription() {
+  return jt`Control the visibility of links to Metabase documentation and Metabase references in your instance. ${(
+    <Popover key="popover" position="top-start">
+      <Popover.Target>
+        <Anchor style={{ userSelect: "none", cursor: "pointer" }}>
+          {t`Learn more`}
+        </Anchor>
+      </Popover.Target>
+      <Popover.Dropdown>
+        <Stack p="md" spacing="sm" maw={420}>
+          <Text size="sm">
+            {t`This affects all links in the product experience (outside of the
```

Comment:
I know HTML doesn't care about new lines, but I run `./bin/i18n/update-translation-template` and end up in the "keys" of the translations
```
#: enterprise/frontend/src/metabase-enterprise/whitelabel/components/MetabaseLinksToggleWidget/MetabaseLinksToggleDescription.tsx:15
msgid ""
"This affects all links in the product experience (outside of the\n"
"admin panel) that point to Metabase.com URLs."
msgstr ""
```

(It was most likely prettier that added them when the strings weren't wrapped in `t`)

Reply:
Good catch once again, I only added `t` around the text, previously the text was formatted like that, but when they're wrapped in `t` they're not supposed to wrap.

Reply:
Fixed in 5ad7468851696211cb1c5edb321545bf6de9adc7